src/components/planNotification/services/planNotification.service.js

```diff
@@ -0,0 +1,83 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-web-app.planNotification').service('PlanNotificationService',
+        function ($translate, PopUpSrv, $q, $window, $log, ENV, $http, AuthService, $location) {
+            'ngInject';
+
+            function _getPlanIdFromUrl() {
+                var search = $location.search();
+                return angular.isDefined(search.planId) ? search.planId : null;
+            }
+
+            function _checkPlanNotification(){
+                var planId = _getPlanIdFromUrl();
+                if (planId) {
+                    var uid = AuthService.getAuth().uid;
+                    var connectStudentToPlanUrl = ENV.myZinkerz + '/plan/connectStudentToPlan';
+                    $http({
+                        method: 'POST',
+                        url: connectStudentToPlanUrl,
+                        data: { planId: planId, uid: uid }
+                    }).then(function successCallback() {
+                        _newPlanNotification({ refObjId: planId });
+                        $log.debug('checkPlanNotification: connectStudentToPlan successful');
+                    }, function errorCallback(err) {
+                        $log.error('checkPlanNotification: error in PlanService.connectStudentToPlan, err: ' + err);
+                    });
+                }
+            }
+
+            function _newPlanNotification(notification) {
+                _showPlanNotificationPopUp().then(function () {
+                    $window.open(ENV.myZinkerz + '?planId=' + notification.refObjId);
```

Comment:
Route directly to: ENV.myZinkerz + '/newPlan"

Discuss with Assaf regarding if we can have the "newPlan" component without a template, only logic...